widget/embedded/src/components/SwapDetails/SwapDetails.tsx

```diff
@@ -319,11 +320,26 @@ export function SwapDetails(props: SwapDetailsProps) {
               <Typography variant="label" size="small" color="neutral700">
                 {requestId}
               </Typography>
-              <IconButton
-                variant="ghost"
-                onClick={handleCopy.bind(null, requestId || '')}>
-                <CopyIcon size={16} color="gray" />
-              </IconButton>
+              <Tooltip
+                container={getContainer()}
+                content={
+                  isCopied
+                    ? i18n.t('Copied To Clipboard')
+                    : i18n.t('Copy Request ID')
+                }
+                open={isCopied || undefined}
+                side="bottom">
+                <IconButton
+                  variant="ghost"
+                  onClick={handleCopy.bind(null, requestId || '')}>
+                  {isCopied ? (
+                    <DoneIcon size={16} color="secondary" />
+                  ) : (
+                    <CopyIcon size={16} color="gray" />
+                  )}
+                </IconButton>
+              </Tooltip>
```

Comment:
The alignment of the tooltip at the bottom of the copy icon does not match the UI design. The `Radix` `Tooltip.Content` component accepts a prop called `alignOffset` that can be combined with the `align` prop to adjust the tooltip's alignment. You can expose this prop through the UI package and use it as needed:
```ts
  <Tooltip
                ...
                alignOffset={-20}
                align="end"
               >
```

Reply:
fixed